book.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'rental'
+require_relative 'person'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, person, self)
+    @rentals << rental
+  end
```

Comment:
- [ ] We would have duplicate `rental` in the rentals array. Note that creating a rental on line 14 already adds itself to the respective arrays of book and person object, so we don't need line 15 anymore. Let's change to this:

  ```
  def add_rental(date, person)
    Rental.new(date, person, self)
  end
  ```